code/find/lib/highlights.js

```diff
@@ -1,98 +1,98 @@
 module.exports = {
   "Mercurial": {
-    "en": "Hot Destination",
-    "ko": "MD의 Pick! 핫 플레이스"
+    "en": { label: "Hot Destination", dialog: "A hot destination is The Mercurial."},
+    "ko": { label: "MD의 Pick! 핫 플레이스", dialog: "A hot destination is The Mercurial."}
```

Comment:
YT for translations: https://sixfivelabs.myjetbrains.com/youtrack/issue/VIV-25643